pytest_trio/_tests/test_hypothesis_interaction.py

```diff
@@ -0,0 +1,21 @@
+import pytest
+from hypothesis import given, strategies as st
+
+
+@given(st.integers())
+@pytest.mark.trio
+async def test_mark_inner(n):
+    pass
+
+
+@pytest.mark.trio
+@given(st.integers())
+async def test_mark_outer(n):
+    pass
+
+
+@pytest.mark.parametrize('y', [1, 2])
+@given(x=st.integers())
+@pytest.mark.trio
+async def test_mark_and_parametrize(x, y):
+    pass
```

Comment:
Same here, but I would make the `parametrize` and `given` produce different types of data just to be sure we don't get `x` and `y` mixed